src/chains/ethereum_1/ethChainState.ts

```diff
@@ -131,26 +131,37 @@ export class EthereumChainState {
     }
   }
 
+  getTransactionHash(signedTransaction: string) {
+    return new Promise((resolve, reject) => {
+      this._web3.eth.sendSignedTransaction(signedTransaction).once('transactionHash', hash => {
+        resolve(hash)
+      })
+    })
+  }
+
   /** Broadcast a signed transaction to the chain */
```

Comment:
This comment could be more helpful. You should ask yourself what does the developer need to know to write her code. I would add a few lines about ConfirmType to explain .None returns a tx hash and .After001 waits until...